prisma/datamodel.prisma

```diff
@@ -2,12 +2,31 @@ type User {
   id: ID! @unique
   email: String! @unique
   password: String!
-  name: String
+  displayName: String
   photoUrl: String
-  description: String
-  createdAt: DateTime!
-  updatedAt: DateTime!
+  statusMsg: String
   friends: [User!]! @relation(name: "Friend")
+  isOnline: Boolean
+  created: DateTime!
+  updated: DateTime!
+}
+
+type Chatroom {
+  id: ID! @unique
+  lastChat: Chat @relation(name: "LastChat")
+  lastChatCnt: Int
+  chats: [Chat!]! @relation(name: "Chat")
+  users: [User!]! @relation(name: "User")
```

Comment:
seems you don't need `@relation` for Chat,

but users in Chatroom need something like `UserMembership` because it is a many-to-many relationship; might be ambiguous to Prisma